user/serializers.py

```diff
@@ -41,15 +41,16 @@ def validate(self, attrs):
             if user:
                 if not user.is_active:
                     msg = _("User account is disabled.")
-                    raise serializers.ValidationError(
-                        msg, code="authorization"
-                    )
+                    raise serializers.ValidationError(msg,
+                                                      code="authorization")
             else:
                 msg = _("Unable to log in with provided credentials.")
-                raise serializers.ValidationError(msg, code="authorization")
+                raise serializers.ValidationError(msg,
+                                                  code="authorization")
         else:
             msg = _("Must include 'username' and 'password'.")
```

Comment:
In your error message, you are asking for 'username' and 'password'. However, your function is expecting 'email' and 'password'. You should update the error message to reflect this.